Makefile

```diff
@@ -9,18 +9,32 @@ IMAGE_NAME := node-feature-discovery
 IMAGE_TAG_NAME := $(VERSION)
 IMAGE_REPO := $(IMAGE_REGISTRY)/$(IMAGE_NAME)
 IMAGE_TAG := $(IMAGE_REPO):$(IMAGE_TAG_NAME)
-
+K8S_NAMESPACE := kube-system
 
 all: image
 
-image:
+image: nfd-master.yaml nfd-worker-daemonset.yaml
 	$(IMAGE_BUILD_CMD) --build-arg NFD_VERSION=$(VERSION) \
 		-t $(IMAGE_TAG) ./
 
+%.yaml: %.yaml.template FORCE
```

Comment:
How about just putting the names of the YAMLs in a variable and creating a phony target for generating them? Would get rid of the `FORCE` target. WDYT?

Reply:
Variables added.

Phony target (yamls) added. (Good idea!)

We won't get rid of `FORCE`, however, because yaml instances depend on both templates and configuration variable values (`IMAGE_TAG` and `K8S_NAMESPACE`). Dropping `FORCE` would easily lead situations like:
- yamls would use older image than the latest build (example: user changed `IMAGE_REGISTRY` for the latest build)
- `make K8S_NAMESPACE=nfd-devel nfd-master.yaml` would report `...is already up to date` yet the yaml would actually contain different namespace.

Reply:
> We won't get rid of `FORCE`

True, my bad. I thought you could nicely generate all yamls with one _simple_ command (like `sed -i`), but, that is not possible because you need to copy files